tests/fixtures/mocked_panda.py

```diff
@@ -47,16 +46,21 @@
 
 # If the test is cancelled half way through then the softioc process isn't always killed
 # Use the unique TEST_PREFIX to ensure this isn't a problem for future tests
-TEST_PREFIX = "TEST-PREFIX-" + str(uuid4())[:4].upper()
 BOBFILE_DIR = Path(__file__).parent.parent / "test-bobfiles"
-TIMEOUT = 10
+TIMEOUT = 100000
+TEST_PREFIX = "TEST_PREFIX"
+
+
+@pytest.fixture
+def new_random_test_prefix():
+    return TEST_PREFIX + "-" + str(uuid4())[:8].upper()
 
 
 @pytest_asyncio.fixture
-def mocked_time_record_updater():
+def mocked_time_record_updater(new_random_test_prefix):
```

Comment:
Would be nice to have the type hints here, or at least in the docstring, as it's no longer a single object